src/main.c

```diff
@@ -54,6 +62,22 @@ inline void screenOn() {
 inline void screenOff() {
     g.pauseScreenOff = 0;
 }
+struct buttonHandler mainButtonHandler = {
```

Comment:
Don't move this block so you can avoid forward declaring the functions.  It also makes it clear if the contents of this struct has changed.